app/src/main/java/org/traccar/client/TrackingController.java

```diff
@@ -100,7 +100,8 @@ public void onPositionUpdate(Position position) {
 
     @Override
     public void onNetworkUpdate(boolean isOnline) {
-        StatusActivity.addMessage(context.getString(R.string.status_connectivity_change));
+        int message = isOnline? R.string.status_network_online: R.string.status_network_offline;
```

Comment:
There is a formatting issue here. You need spaces on both sides around `?` and `:`.

Reply:
Fixed